src/modules/Shared/Components/AuthLayout/AuthLayout.tsx

```diff
@@ -1,11 +1,90 @@
-import { Grid2 } from "@mui/material";
-import React from "react";
+import { Box, Grid2, Typography } from "@mui/material";
+import { Outlet, useLocation } from "react-router-dom";
+import RegisterImg from "../../../../assets/bg-register.png";
+import LoginImg from "../../../../assets/bg-login.png";
+import ForgetImg from "../../../../assets/bg-forget.png";
+import Logo from "../Logo/Logo";
 
 export default function AuthLayout() {
+  const { pathname } = useLocation();
+  const renderImg = () => {
```

Comment:
It is better when we have logic that is not specific to react to make it outside of the component to avoid re-creating the function when the component re-render. We can make the function outside of the component and take the pathname as a parameter.